The original spin no longer compile with Java SDK 11+

Compiling the original spin caused following failures:

```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] Source option 1.4 is no longer supported. Use 6 or later.
[ERROR] Target option 1.4 is no longer supported. Use 1.6 or later.
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------

```

--- a/.gitignore
+++ b/.gitignore
@@ -1,0 +1,4 @@
+target
+.project
+.classpath
+.settings

--- a/pom.xml
+++ b/pom.xml
@@ -4,9 +4,9 @@
   <groupId>spin</groupId>
   <artifactId>spin</artifactId>
   <packaging>jar</packaging>
-  <version>1.5</version>
+  <version>1.5b1</version>
   <name>Spin</name>
-  <url>http://spin.sourceforge.net</url>
+  <url>https://github.com/yuansc/spin</url>
   <description>Transparent threading solution for non-freezing Swing applications.</description>
 
   <licenses>

--- a/pom.xml
+++ b/pom.xml
@@ -18,8 +18,8 @@
   </licenses>
 
   <issueManagement>
-    <system>Sourceforge</system>
-    <url>http://sourceforge.net/tracker?group_id=3636</url>
+    <system>GitHub.com</system>
+    <url>https://github.com/yuansc/spin/issues</url>
   </issueManagement>
   
   <developers>
@@ -33,8 +33,17 @@
         <role>Developer</role>
       </roles>
     </developer>         
+    <developer>
+      <id>yuansc</id>
+      <name>Scott Ling-Hsuing Yuan</name>
+      <email>yuansc@yahoo.com</email>
+      <roles>
+        <role>Developer</role>
+      </roles>
+    </developer>         
   </developers>
    
+  <!--
   <mailingLists>
     <mailingList>
       <name>spin-developer</name>
@@ -50,6 +59,7 @@
     <developerConnection>scm:extssh:spin.cvs.sourceforge.net:/cvsroot/spin</developerConnection>
     <url>http://spin.cvs.sourceforge.net/spin/</url>
   </scm> 
+  -->
    
   <dependencies>
     <dependency>
@@ -125,8 +135,8 @@
       <plugin>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.4</source>
-          <target>1.4</target>
+          <source>11</source>
+          <target>11</target>
         </configuration>
       </plugin>
       <plugin>

